proto/ssn/ocrservice/v1/ocrservice.proto

```diff
@@ -13,6 +13,7 @@ option java_outer_classname = "OcrServiceProto";
 service OcrService {
   rpc OcrScanImage(OcrScanImageRequest) returns (OcrScanImageResponse);
   rpc GetTextAnnotation(GetTextAnnotationRequest) returns (GetTextAnnotationResponse);
+  rpc GetTextAnnotations(GetTextAnnotationsRequest) returns (GetTextAnnotationsResponse);
```

Comment:
you forgot stream as in https://github.com/e-conomic/vmlapis/blob/master/proto/ssn/pdfservice/v1/pdfservice.proto